src/App.js

```diff
@@ -2,13 +2,15 @@ import React, { Component } from 'react';
 import './App.css';
 import Grid from './Grid.js';
 import LastValue from './LastValue.js';
+import LastFiveValues from './LastFiveValues.js';
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
       nextSelectedValue: null,
```

Comment:
Now that we have `lastFiveSelectedValues`, we should get rid of this, and just access `lastFiveSelectedValues[-1]` wherever we use this.